src/SimSpace/SimFormat.hs

```diff
@@ -33,16 +33,23 @@ type Parser = Parsec Void String
 -- The key type is carefully chosen so `Map.toList` puts the package imports at the top,
 -- then qualified imports at the bottom,
 -- and then sorts by module name.
+-- Order of the key fields:
+--    importStmtPackageName
+--    importStmtQualified
+--    importStmtModuleName
+--    importStmtAlias
+--    importStmtCommented
 newtype SortedImportStmts = SortedImportStmts
-  { unSortedImportStmts :: Map (Maybe String, Bool, String, Maybe String) SortedImportList
+  { unSortedImportStmts :: Map (Maybe String, Bool, String, Maybe String, Bool) SortedImportList
```

Comment:
Commented imports are placed last when there's a conflict. Otherwise, they are placed in alphabetical order.